cloud/deploys/staging_azure/variables.tf

```diff
@@ -38,3 +38,8 @@ variable "sender_email_address" {
   type        = string
   description = "Email address that emails will be sent from"
 }
+
+variable "staging_hostname" {
+  type        = string
+  description = "URL for staging"
```

Comment:
suggestion (blocking): please clarify URL or hostname. is the protocol included?